### DISCUSSION TITLE
No audio (yet) during training?

### DISCUSSION THREAD

**Discussion by JulienVincenot** [acids-ircam/RAVE#159] [Q&A] [answered]
Hi, 

I'm not very far in my first training attempt, around 1500 epochs this morning on a quite small dataset (I'm not sure how long, GDrive is always awful to get file infos). 
I wondered, the audio section of Tensorboard still returns errors for all players. Is that to be expected?...

By the way I'm still quite new with Colab, Tensorboard only seems to display data up to the last version, not the current segment I'm running. Curious if that's normal as well, since I got no error and Rave is still training (happily?).

Thanks !

Julien

![Capture d’écran 2023-01-27 à 2 40 08 PM](https://user-images.githubusercontent.com/18226950/215025596-d7d77f77-c9bf-4f68-94fe-332af89aca80.png)

**Comment by caillonantoine** [comment, marked as answer]
Hi Julien, unfortunately you cannot preview audio from Safari, you should use chrome or firefox :)

**Comment by JulienVincenot** [comment]
Oooooh I should have known, thanks Antoine !
I thought about it but didn't think it would be such a hard task to play a pcm file (I assume) 😅

Again I'm still struggling with Colab, do you think that's okay to switch browser midway through a training project?
I'm a bit afraid to lose everything, every time Colab times out I reinstall everything hoping not to do any mistake.

Merci encore pour la réponse rapide 🙏

**Reply by caillonantoine** [reply]
Avec plaisir ;)

**Reply by branlsnyder** [reply]
Hi Julien, in my experience (was using the Rave Colab notebooks in September 2022), I needed to keep the browser window open. If I needed to use the browser for something else, I would open another window. Occasionally,(but not always...) Colab would ask "if I am still there", and I would need to hit yes to keep my training resuming.

**Reply by JulienVincenot** [reply]
Hey Brandon (long time no see 😄)
Yes I'm got the prompt today, I guess that's why my connection dropped last night. Luckily was able to restart easily.
It's a bit cumbersome, I read somewhere that Colab Pro+ allows to close the browser tab and reload later, I may switch to that one in a bit...

**Reply by branlsnyder** [reply]
yes long time no see :) 
indeed, it was cumbersome for me. i had colab pro (not + ). I also tried some cheap AWS instances, but colab is the most price efficient option, if you don't mind these things

**Reply by JulienVincenot** [reply]
Well I do mind a bit (feels like losing all my money at a slot machine ...)
but I'm a pre M1 mac user so not much choice for a while
What config are you working with now?

**Comment by JulienVincenot** [comment]
@caillonantoine 
I managed to migrate my run from Safari to Chrome, luckily I didn't break anything and the training is still running.
This time I can refresh TensorBoard without crashing it, which is cool I don't need to interrupt the run to get visual and audio feedback anymore...!

So now I do get sound, and I'd like to know what I should be careful about here. Basically I trained on a rather small database (2 hours tops) with very heterogeneous composed sequences of sounds.
 
What I hear in TensorBoard is a kind of chopped medley of several passages that I can recognize (well it's from my music), every 1 or 2 seconds the preview changes to another passage, seems "random". They are kind of identical to the originals (I think I don't hear much variations) but the quality is quite good. 

Does this seem okay, or it's just showing my dataset is too small or not diverse enough?

Thanks !
